docs/topics/Gateway.md

```diff
@@ -903,18 +903,18 @@ Sent when an invite is deleted.
 
 #### Message Create
 
-Sent when a message is created. The inner payload is a [message](#DOCS_RESOURCES_CHANNEL/message-object) object.
+Sent when a message is created. The inner payload is a [message](#DOCS_RESOURCES_CHANNEL/message-object) object. If this message is a reply, it is guaranteed to have a `referenced_message` field.
```

Comment:
It is not guaranteed.  Only up to 5 messages who are replying to _old_ messages are loaded.  So _most_ of the time will always be there, but if 50 people reply to the first 50 messages in the channel, only the first 5 will be there

Reply:
Oh, also if the referenced message was deleted we obviously won't return it here either

Reply:
wouldn't those not apply to the message create gateway event?

Reply:
Gonna just remove this then.